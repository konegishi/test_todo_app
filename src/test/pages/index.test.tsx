import { render } from '@testing-library/react';
import React from 'react';
import Home from '../../pages';

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Home />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
