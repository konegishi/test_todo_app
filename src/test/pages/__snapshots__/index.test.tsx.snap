// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="absolute w-full h-full bg-gray-200"
    >
      <div
        class="flex content-center items-center justify-center h-full"
      >
        <div
          class="bg-white rounded shadow p-4 w-11/12"
        >
          <h2
            class="text-xl text-center border-b mb-4 pb-4 text-gray-800 font-bold"
          >
            ToDo App
          </h2>
          <div
            class="sbui-auth"
          >
            <div
              class="sbui-space-col sbui-space-y-8"
            >
              <div
                class="sbui-space-col sbui-space-y-8"
              >
                <div
                  class="sbui-space-col sbui-space-y-4"
                >
                  <span
                    class="sbui-typography-text sbui-auth-label sbui-typography-text-secondary"
                  >
                    Sign in with
                  </span>
                  <div
                    class="sbui-space-row sbui-space-x-2"
                  >
                    <div
                      style="flex-grow: 1;"
                    >
                      <span
                        class="sbui-btn-container sbui-btn--w-full"
                      >
                        <button
                          class="sbui-btn sbui-btn-default sbui-btn--w-full sbui-btn-container--shadow sbui-btn--xlarge flex items-center sbui-btn--text-align-center"
                        >
                          <svg
                            aria-hidden="true"
                            data-icon="google"
                            data-prefix="fab"
                            focusable="false"
                            role="img"
                            viewBox="0 0 488 512"
                            width="21"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z"
                              fill="currentColor"
                            />
                          </svg>
                        </button>
                      </span>
                    </div>
                    <div
                      style="flex-grow: 1;"
                    >
                      <span
                        class="sbui-btn-container sbui-btn--w-full"
                      >
                        <button
                          class="sbui-btn sbui-btn-default sbui-btn--w-full sbui-btn-container--shadow sbui-btn--xlarge flex items-center sbui-btn--text-align-center"
                        >
                          <svg
                            aria-hidden="true"
                            class="svg-inline--fa fa-github fa-w-16"
                            data-icon="github"
                            data-prefix="fab"
                            focusable="false"
                            role="img"
                            viewBox="0 0 496 512"
                            width="21"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                              fill="currentColor"
                            />
                          </svg>
                        </button>
                      </span>
                    </div>
                  </div>
                </div>
                <div
                  class="sbui-divider sbui-divider--center"
                  role="seperator"
                >
                  <span>
                    or continue with
                  </span>
                </div>
              </div>
              <form
                id="auth-sign-in"
              >
                <div
                  class="sbui-space-col sbui-space-y-6"
                >
                  <div
                    class="sbui-space-col sbui-space-y-3"
                  >
                    <div>
                      <div
                        class="sbui-formlayout sbui-formlayout--medium sbui-formlayout--responsive"
                      >
                        <div
                          class="sbui-space-row sbui-space-x-2 sbui-formlayout__label-container-horizontal"
                        >
                          <label
                            class="sbui-formlayout__label"
                          >
                            Email address
                          </label>
                        </div>
                        <div
                          class="sbui-formlayout__content-container-horizontal"
                        >
                          <div
                            class="sbui-input-container"
                          >
                            <input
                              autocomplete="email"
                              class="sbui-input sbui-input--with-icon sbui-input--medium"
                              type="text"
                              value=""
                            />
                            <div
                              class="sbui-input-icon-container"
                            >
                              <svg
                                class="sbui-icon "
                                fill="none"
                                height="21"
                                stroke="#666666"
                                stroke-linecap="round"
                                stroke-linejoin="round"
                                viewBox="0 0 24 24"
                                width="21"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"
                                />
                                <polyline
                                  points="22,6 12,13 2,6"
                                />
                              </svg>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div>
                      <div
                        class="sbui-formlayout sbui-formlayout--medium sbui-formlayout--responsive"
                      >
                        <div
                          class="sbui-space-row sbui-space-x-2 sbui-formlayout__label-container-horizontal"
                        >
                          <label
                            class="sbui-formlayout__label"
                          >
                            Password
                          </label>
                        </div>
                        <div
                          class="sbui-formlayout__content-container-horizontal"
                        >
                          <div
                            class="sbui-input-container"
                          >
                            <input
                              autocomplete="current-password"
                              class="sbui-input sbui-input--with-icon sbui-input--medium"
                              type="password"
                              value=""
                            />
                            <div
                              class="sbui-input-icon-container"
                            >
                              <svg
                                class="sbui-icon "
                                fill="none"
                                height="21"
                                stroke="#666666"
                                stroke-linecap="round"
                                stroke-linejoin="round"
                                viewBox="0 0 24 24"
                                width="21"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M21 2l-2 2m-7.61 7.61a5.5 5.5 0 1 1-7.778 7.778 5.5 5.5 0 0 1 7.777-7.777zm0 0L15.5 7.5m0 0l3 3L22 7l-3-3m-3.5 3.5L19 4"
                                />
                              </svg>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="sbui-space-col sbui-space-y-6"
                  >
                    <div
                      class="sbui-space-row sbui-space-x-2"
                      style="justify-content: space-between;"
                    >
                      <div
                        class="sbui-checkbox-container sbui-checkbox-container--medium"
                      >
                        <input
                          class="sbui-checkbox"
                          id="remember_me"
                          name="remember_me"
                          type="checkbox"
                          value="remember_me"
                        />
                        <div
                          class="sbui-checkbox__label-container"
                        >
                          <label
                            class="sbui-checkbox__label-container__label"
                            for="remember_me"
                          >
                            <span
                              class="sbui-checkbox__label-container__label__span"
                            >
                              Remember me
                            </span>
                          </label>
                        </div>
                      </div>
                      <a
                        class=" sbui-typography-link"
                        href="#auth-forgot-password"
                        rel="noopener noreferrer"
                        target="_blank"
                      >
                        Forgot your password?
                      </a>
                    </div>
                    <span
                      class="sbui-btn-container sbui-btn--w-full"
                    >
                      <button
                        class="sbui-btn sbui-btn-primary sbui-btn--w-full sbui-btn-container--shadow sbui-btn--large sbui-btn--text-align-center"
                        type="submit"
                      >
                        <svg
                          class="sbui-icon "
                          fill="none"
                          height="21"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          viewBox="0 0 24 24"
                          width="21"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <rect
                            height="11"
                            rx="2"
                            ry="2"
                            width="18"
                            x="3"
                            y="11"
                          />
                          <path
                            d="M7 11V7a5 5 0 0 1 10 0v4"
                          />
                        </svg>
                        <span>
                          Sign in
                        </span>
                      </button>
                    </span>
                  </div>
                  <div
                    class="sbui-space-col sbui-space-y-2"
                    style="text-align: center;"
                  >
                    <a
                      class=" sbui-typography-link"
                      href="#auth-sign-up"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      Don't have an account? Sign up
                    </a>
                  </div>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
